### PR TITLE
Update logout page style and functionality to match other pages on auth server (#1448)

### PR DESCRIPTION
**Pull request by martsiv**
- moved logout page to AuthCommon
- added the js code for it in the corresponding directory
- added changes for logout page in localization file
- added changes to the css file

### CODE REVIEW

File: OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj [hunk 1]
```diff
@@ -37,10 +37,19 @@
             <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
         </PackageReference>
         <Content Remove="stylecop.json" />
+        <Content Remove="wwwroot\js\logout.js" />
```
Comment (DmyMi) [inline OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj]: we already have watch on the wwwroot folder, please remove this

File: OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj [hunk 2]
```diff
@@ -37,10 +37,19 @@
...
         <AdditionalFiles Include="stylecop.json" />
     </ItemGroup>
 
     <ItemGroup>
         <Watch Include="wwwroot\**" />
     </ItemGroup>
+
```
Comment (DmyMi) [inline OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj]: and these two `ItemGroup`s

**Review by DmyMi — CHANGES_REQUESTED** [review]

File: OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj (hunk 1 above)
Comment (martsiv) [reply OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj]: Was removed

File: OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj (hunk 2 above)
Comment (martsiv) [reply OutOfSchool/OutOfSchool.AuthCommon/OutOfSchool.AuthCommon.csproj]: Were removed two groups

**Review by VadymLevkovskyi — APPROVED** [review]

**Review by DmyMi — APPROVED** [review]

**Pull request merged**